atlite/datasets/mrel_wave.py

```diff
@@ -0,0 +1,108 @@
+import logging
+
+import numpy as np
+import xarray as xr
+from rasterio.warp import Resampling
+
+from atlite.gis import regrid
+
+logger = logging.getLogger(__name__)
+
+crs = 4326
+dx = 0.0625
+dy = 0.04
+
+features = {"wave_height": ["wave_height"], "wave_period": ["wave_period"]}
+
+
+def _rename_and_clean_coords(ds):
+    """
+    Rename 'longitude' and 'latitude' columns to 'x' and 'y' and fix roundings.
+
+    Optionally (add_lon_lat, default:True) preserves latitude and
+    longitude columns as 'lat' and 'lon'.
+    """
+    ds = ds.rename({"longitude": "x", "latitude": "y"})
+
+    ds = ds.assign_coords(
+        x=np.round(ds.x.astype(float), 5), y=np.round(ds.y.astype(float), 5)
+    )
+
+    return ds
+
+
+def get_data_wave_height(ds):
+    ds = ds.rename({"hs": "wave_height"})
+    ds["wave_height"] = ds["wave_height"].clip(min=0.0)
+
+    return ds
+
+
+def get_data_wave_period(ds):
+    ds = ds.rename({"tp": "wave_period"})
+    # ds["wave_period"] = (1 / ds["wave_period"])
+    ds["wave_period"] = ds["wave_period"].clip(min=0.0)
+
+    return ds
```

Comment:
Keep the process the same as in `era5.py` - split this into data retrieval and then sanitisation and call the sanitisation function only if requested in `get_data` (you can copy most of the functionality directly over from `era5.py`)